radar/languages-frameworks.js

```diff
@@ -100,17 +100,21 @@ const content = [
   },
   {
     name: 'Cypress',
-    ring: 'Hold',
+    ring: 'Adopt',
```

Comment:
Ik heb er niets op tegen om hem weer op `adopt` te zetten, maar dan heb ik wel het idee dat wij iets meer moeten zeggen dan dat we nu aanpassen